### PR TITLE
Do not crash on missing multipath device wwn.

### PR DESCRIPTION
**Pull request by rvykydal**
Blivet sets the wwn attribute value to None if it doesn't find
udev ID_WWN_WITH_EXTENSION value.

Resolves: rhbz#1666242

### CODE REVIEW

**Comment by rvykydal** [comment]
I tested the patch with multipath over iSCSI targets with blivet library patched to return None value for multipath device wwn attribute.

File: pyanaconda/ui/tui/spokes/storage.py [hunk 1]
```diff
@@ -263,7 +263,7 @@ def _format_disk_info(self, disk):
                 isinstance(disk, iScsiDiskDevice) or
                 isinstance(disk, FcoeDiskDevice)):
             if hasattr(disk, "wwn"):
-                disk_attrs.append(disk.wwn)
+                disk_attrs.append(disk.wwn or "")
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/ui/tui/spokes/storage.py]: Good point, I'll fix that.

File: pyanaconda/ui/gui/spokes/advanced_storage.py [hunk 2]
```diff
@@ -310,7 +310,7 @@ def _filter_func(self, device):
         elif filterBy == self.SEARCH_TYPE_INTERCONNECT:
             return device.bus == self._icCombo.get_active_text()
         elif filterBy == self.SEARCH_TYPE_WWID:
-            return self._wwidEntry.get_text() in device.wwn
+            return self._wwidEntry.get_text() in (device.wwn or "")
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/ui/gui/spokes/advanced_storage.py]: Ok.

**Comment by rvykydal** [comment]
No need to port this to master. We are already using "" there.

**Pull request merged**